tx.go

```diff
@@ -201,11 +194,20 @@ func (tx *Tx) Commit() (err error) {
 		countFlag = CountFlagDisabled
 	}
 
-	buff := cachePool.Get().(*bytes.Buffer)
-	defer func() {
-		buff.Reset()
-		cachePool.Put(buff)
-	}()
+	var txSize int64
```

Comment:
Could you help add test cases for your change logic?

Reply:
Okay, I'll do some unit tests.